global-raw-tokens/src/main/java/com/orange/ouds/tokens/BorderRawTokens.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.tokens.raw
+
+object BorderRawTokens {
+    const val borderBase = 4
```

Comment:
We could remove the `border` prefix because it's redundant with `BorderRawTokens` when it is used. However we can't do this with opacity because opacity token names are `opacityX`. I don't know if there is an other solution.

Reply:
These tokens will be generated by Style Dictionary so the name will be the name used in Figma. I kept the name provided by Figma.